requirements.txt

```diff
@@ -15,3 +15,4 @@ setuptools
 appdirs>=1.4.3
 dataclasses; python_version < '3.7'
 typing_extensions; python_version < '3.10'
+httpx>=0.22.0  # 0.23.0 dropped Python 3.6; we can upgrade once we drop it too
```

Comment:
what do you think about having this as an extra?

Reply:
Seems reasonable, since it's not required for normal operation of `wandb`!

How about in `requirements_dev.txt`? Can I put `httpx` and `respx` in there, or should that be replaced by extras too? (Making them extras would mildly complicate the testing of `InternalApi` -- I think I'd need to move all the async tests into their own file so I could use `pytest.importorskip`, but I like keeping the sync/async tests next to each other, to avoid drift.)

Reply:
no dev i don't think we care, since it is only for our dev env.